integrated_channels/sap_success_factors/utils.py

```diff
@@ -0,0 +1,86 @@
+"""
+Utilities for Enterprise Integrated Channel SAP SuccessFactors.
+"""
+from __future__ import absolute_import, unicode_literals
+
+import datetime
+
+from enterprise.lms_api import parse_lms_api_datetime
+from integrated_channels.integrated_channel.course_metadata import BaseCourseExporter
+from integrated_channels.sap_success_factors.models import (
+    SAPSuccessFactorsGlobalConfiguration
+)
+
+
+UNIX_EPOCH = datetime.datetime(1970, 1, 1)
+UNIX_MIN_DATE_STRING = '1970-01-01T00:00:00Z'
+UNIX_MAX_DATE_STRING = '2038-01-19T03:14:07Z'
+
+
+class SapCourseExporter(BaseCourseExporter):
+    """
+    Class to provide data transforms for SAP SuccessFactors course export task.
+    """
+
+    data_transform = {
+        'courseID': lambda x: x['key'],
+        'providerID': lambda x: SAPSuccessFactorsGlobalConfiguration.current().provider_id,
+        'status': lambda x: 'ACTIVE' if x['availability'] == 'Current' else 'INACTIVE',
+        'title': lambda x: [
+            {
+                'locale': 'English',
+                'value': x['title']
+            },
+        ],
+        'description': lambda x: [
+            {
+                'locale': 'English',
+                'value': x['full_description'] or '',
+            },
+        ],
+        'thumbnailURI': lambda x: (x['image']['src'] or ''),
+        'content': lambda x: [
+            {
+                'providerID': SAPSuccessFactorsGlobalConfiguration.current().provider_id,
+                'launchURL': x['marketing_url'] or '',
```

Comment:
Sounds good. The whole point of this structure is to make it easy to write/change transforms for individual fields.